core/src/main/scala/org/bitcoins/core/script/ScriptProgram.scala

```diff
@@ -94,7 +94,11 @@ case class ExecutionInProgressScriptProgram(
     altStack: List[ScriptToken],
     flags: Seq[ScriptFlag],
     lastCodeSeparator: Option[Int])
-    extends StartedScriptProgram
+    extends StartedScriptProgram {
+  def addFlags(newFlags: Seq[ScriptFlag]): ExecutionInProgressScriptProgram = {
```

Comment:
I agree with @Christewart s comment that this should probably be called `replaceFlags`, but additionally I think that we should also have a `removeFlags` method that does the equivalent of passing in an empty `newFlags` here since this does seem to be the most common use case by far

Reply:
That makes a lot of sense, I'll do that instead